samtranslator/model/sam_resources.py

```diff
@@ -161,6 +162,7 @@ class SamFunction(SamResourceMacro):
     EphemeralStorage: Optional[Dict[str, Any]]
     AutoPublishAlias: Optional[Intrinsicable[str]]
     AutoPublishCodeSha256: Optional[Intrinsicable[str]]
+    AutoPublishAliasAdditionalProperties: Optional[Intrinsicable[bool]]
```

Comment:
Don't resolve intrinsics on new properties, it's handled by `AWS::LanguageExtensions`. So this isn't `Intrinsicable`.